Jenkinsfile

```diff
@@ -1,95 +1,113 @@
 #!groovy
 
-// Project Settings for Deployment
-String PROJECT_DISPLAY = "iiif-auth-proxy"
-String PROJECT_NAME = "iiif-auth-proxy"
-String CONTAINER_DIR = "."
-String PRODUCTION_BRANCH = "master"
-String ACCEPTANCE_BRANCH = "development"
-String INFRASTRUCTURE = 'thanos'
-String PLAYBOOK = 'deploy.yml'
+def PROJECT_NAME = "iiif-auth-proxy"
+def PLAYBOOK = 'deploy.yml'
+def SLACK_CHANNEL = '#waarnemingen-deployments'  # TODO: Change to a more generic deployment channel
+def SLACK_MESSAGE = [
+    "title_link": BUILD_URL,
+    "fields": [
+        ["title": "Project","value": PROJECT_NAME],
+        ["title":"Branch", "value": BRANCH_NAME, "short":true],
+        ["title":"Build number", "value": BUILD_NUMBER, "short":true]
+    ]
+]
 
-String IMAGE_NAME = "docker-registry.data.amsterdam.nl/datapunt/${PROJECT_NAME}:${env.BUILD_NUMBER}"
-String BRANCH = "${env.BRANCH_NAME}"
 
-image = 'initial value'
+pipeline {
+    agent any
 
-def tryStep(String message, Closure block, Closure tearDown = null) {
-    try {
-        block();
+    options {
+        timeout(time: 1, unit: 'HOURS')
     }
-    catch (Throwable t) {
-        if (BRANCH == "${PRODUCTION_BRANCH}") {
-            slackSend message: "${env.JOB_NAME}: ${message} failure ${env.BUILD_URL}", channel: '#ci-channel', color: 'danger'
-        }
-        throw t;
-    }
-    finally {
-        if (tearDown) {
-            tearDown();
-        }
-    }
-}
 
-node {
-    // Get a copy of the code
-    stage("Checkout") {
-        checkout scm
+    environment {
+        SHORT_UUID = sh( script: "uuidgen | cut -d '-' -f1", returnStdout: true).trim()
+        COMPOSE_PROJECT_NAME = "${PROJECT_NAME}-${env.SHORT_UUID}"
+        VERSION = env.BRANCH_NAME.replace('/', '-').toLowerCase().replace(
+            'master', 'latest'
+        )
+        IS_RELEASE = "${env.BRANCH_NAME ==~ "release/.*"}"
```

Comment:
Dit hebben we bij veel projecten eruit gehaald, master naar acceptatie, tags naar prod.
Zit dit hier bewust nog zo in ?